### DISCUSSION TITLE
Schnellsuche - Information über weitere Treffer

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#119] [Ideas]
Wenn in der Schnellsuche nach einem Begriff gesucht wird, werden derzeit 5 Treffer dargestellt. 
Man weiß aber nicht, ob das alle Treffer sind. 
Wäre eine Information über die Anzahl aller Treffer möglich?
Und ergänzend dazu ein Schalter für die Darstellung weiterer Treffer?

**Comment by gis-eni** [comment]
Ist nicht vorgesehen. Die Schnellsuche muss schnell gehen (quasi in Echtzeit). Darum werden auch nur einmal 5 Treffer abgeholt.
Du kannst allerdings auf die Lupe klicken oder ENTER. Dann werden alle passenden Ergebnisse gesucht.

**Comment by gis-eni** [comment]
Eventuell hinweis (Enter oder auf die Lupe klicken)

**Reply by gis-eni** [reply]
Der Hinweis kommt nicht, aber:

im CMS kann man für jeden Suchdienst angeben, wie viele Ergebnisse abgeholt werden:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/f1ea6284-8eeb-4af3-a618-3f3b0f9c00d3)

So viele Ergebnis kommen vom diesen Suchdienst zurück. Allerdings können in einer Karte mehrere Suchdienste eingebunden sein. Darum werden die Ergebnisse zuerst von den einzelnen Suchdiensten abgeholt und dann noch einmal zusammengestoppelt und gekürzt (bisher immer auf die besten 5 Ergebnisse)

Um hier den Wert zu erhöhen, kann man ab jetzt in der `api.config` noch folgende `section` einbauen:

``` xml
   <section name="quick-search">
      <add key="max-result-items" value="15" />
   </section>
```
